decentraland/helpers/decentraland.js

```diff
@@ -0,0 +1,156 @@
+// Note: Functions that will deal with fetch and prepare data from Decentraland contracts
+import { addressesAreEqual, getContracts } from ".";
+import DecentralandUtils from "tasit-sdk/dist/helpers/DecentralandUtils";
+
+import AssetTypes from "@constants/AssetTypes";
+const { ESTATE, PARCEL } = AssetTypes;
+
+// Note: Returns a list of Promises
+export const getAssetsOf = async address => {
+  const decentralandUtils = new DecentralandUtils();
+  const { getAssetsOf: getLandOf } = decentralandUtils;
+
+  const contracts = getContracts();
+  const { estateContract, landContract } = contracts;
+
+  const listOfLand = await getLandOf(address);
+  const assets = [];
+
+  for (let land of listOfLand) {
+    const { id: assetId, nftAddress } = land;
+
+    let asset = _generateAssetFromId(
+      estateContract,
+      landContract,
+      assetId,
+      nftAddress
+    );
+
+    assets.push(asset);
+  }
+
+  return assets;
+};
+
+// Note: Returns a list of Promises
+export const getAllAssetsForSale = async () => {
+  const decentralandUtils = new DecentralandUtils();
+  const { getAllAssetsForSale: getAllOpenSellOrders } = decentralandUtils;
+
+  const openSellOrders = await getAllOpenSellOrders();
+
+  const contracts = getContracts();
+  const { estateContract } = contracts;
+
+  const estatesForSale = [];
+  for (let order of openSellOrders) {
+    const { nftAddress } = order;
+    const isEstate = addressesAreEqual(nftAddress, estateContract.getAddress());
+    if (isEstate) estatesForSale.push(order);
+  }
+
+  const assetsForSale = [];
+  // Note: Getting only the first 10 assets for now
+  // See more: https://github.com/tasitlabs/tasit/issues/155
+  const listSize = 10;
+  for (let order of estatesForSale.slice(0, listSize)) {
+    let assetForSalePromise = _toAssetForSale(order);
+    assetsForSale.push(assetForSalePromise);
+  }
+
+  return assetsForSale;
+};
+
+const _toAssetForSale = async sellOrder => {
```

Comment:
Oh, I've forgotten our design decision about it.
Even `_getAssetsOf` looks like that could be useful for other parts of the app, I've moved to `MyAssetsScreen.js` too since it's the only component that is using it. Agree?

Reply:
No worries. Yep, that makes sense to me. Thanks!

Reply:
Feel free to resolve this

Reply:
Resolving this.